crates/bevy_render/src/draw.rs

```diff
@@ -48,18 +48,21 @@ pub enum RenderCommand {
 
 #[derive(Debug, Clone, Reflect)]
 #[reflect(Component)]
-pub struct Visible {
-    pub is_visible: bool,
-    // TODO: consider moving this to materials
-    pub is_transparent: bool,
-}
+pub struct Visible;
 
 impl Default for Visible {
     fn default() -> Self {
-        Visible {
-            is_visible: true,
-            is_transparent: false,
-        }
+        Visible
+    }
+}
+
+#[derive(Debug, Clone, Reflect)]
+#[reflect(Component)]
+pub struct Transparent;
```

Comment:
Doc strings on the difference between Transparent and Visible would be nice.

Reply:
what are doc strings here?
just normal comments or something special?

Reply:
[Doc Comments](https://doc.rust-lang.org/rust-by-example/meta/doc.html#doc-comments)

Reply:
Also done.